crates/wasi-nn/src/testing.rs

```diff
@@ -103,10 +103,19 @@ fn check_openvino_artifacts_are_available() -> Result<()> {
             println!("> using cached artifact: {}", local_path.display())
         }
     }
+
+    // Download the ONNX mobilenet model
+    let remote_url = "https://github.com/onnx/models/raw/bec48b6a70e5e9042c0badbaafefe4454e072d08/validated/vision/classification/mobilenet/model/mobilenetv2-7.onnx?download=";
+    let local_path = artifacts_dir.join("model.onnx");
+    if !local_path.is_file() {
+        download(&remote_url, &local_path).with_context(|| "unable to retrieve test artifact")?;
+    } else {
+        println!("> using cached artifact: {}", local_path.display())
+    }
     Ok(())
 }
 
-#[cfg(feature = "winml")]
+#[cfg(all(feature = "winml", target_os = "windows"))]
```

Comment:
Yeah, I think all these additions of `target_os = "windows"` are necessary; thanks for catching that.